parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -312,6 +312,7 @@ ensureKubelet() {
     KUBELET_RUNTIME_CONFIG_SCRIPT_FILE=/opt/azure/containers/kubelet.sh
     wait_for_file 1200 1 $KUBELET_RUNTIME_CONFIG_SCRIPT_FILE || exit $ERR_FILE_WATCH_TIMEOUT
     systemctlEnableAndStart kubelet || exit $ERR_KUBELET_START_FAIL
+    systemctl --no-pager -l status kubelet 2>&1
```

Comment:
Did you get a chance to test this? I'm thinking about moving this to the end of cse execution and let the kubelet run for a while so we get a full log of kubelet?

Reply:
I'm going to test it in a private branch and see what the log looks like

Reply:
moved this to end, and tested in standalone, I can see the logs in ARE